views/islandora_usage_stats_collections.views_default.inc

```diff
@@ -0,0 +1,145 @@
+  /* Field: Islandora Usage Stats: Label */
+  $handler->display->display_options['fields']['label']['id'] = 'label';
+  $handler->display->display_options['fields']['label']['table'] = 'islandora_usage_stats_objects';
+  $handler->display->display_options['fields']['label']['field'] = 'label';
+  $handler->display->display_options['fields']['label']['relationship'] = 'pid_id';
+  /* Field: Islandora Usage Stats: PID */
+  $handler->display->display_options['fields']['pid']['id'] = 'pid';
+  $handler->display->display_options['fields']['pid']['table'] = 'islandora_usage_stats_objects';
+  $handler->display->display_options['fields']['pid']['field'] = 'pid';
+  $handler->display->display_options['fields']['pid']['relationship'] = 'pid_id';
+  $handler->display->display_options['fields']['pid']['element_label_colon'] = FALSE;
+  /* Field: COUNT(DISTINCT Islandora Usage Stats: Log ID) */
+  $handler->display->display_options['fields']['id']['id'] = 'id';
+  $handler->display->display_options['fields']['id']['table'] = 'islandora_usage_stats_object_access_log';
+  $handler->display->display_options['fields']['id']['field'] = 'id';
+  $handler->display->display_options['fields']['id']['group_type'] = 'count_distinct';
+  /* Contextual filter: Islandora Usage Stats: PID */
+  $handler->display->display_options['arguments']['pid']['id'] = 'pid';
+  $handler->display->display_options['arguments']['pid']['table'] = 'islandora_usage_stats_objects';
+  $handler->display->display_options['arguments']['pid']['field'] = 'pid';
+  $handler->display->display_options['arguments']['pid']['relationship'] = 'collection';
+  $handler->display->display_options['arguments']['pid']['default_action'] = 'default';
+  $handler->display->display_options['arguments']['pid']['default_argument_type'] = 'raw';
+  $handler->display->display_options['arguments']['pid']['default_argument_options']['index'] = '2';
+  $handler->display->display_options['arguments']['pid']['summary']['number_of_records'] = '0';
+  $handler->display->display_options['arguments']['pid']['summary']['format'] = 'default_summary';
+  $handler->display->display_options['arguments']['pid']['summary_options']['items_per_page'] = '25';
+  $handler->display->display_options['arguments']['pid']['limit'] = '0';
+
+  /* Display: Block */
+  $handler = $view->new_display('block', 'Block', 'usage_stats');
+  $handler->display->display_options['block_description'] = 'Usage Stats for Collections';
+  $translatables['test'] = array(
+    t('Master'),
+    t('Usage Stats'),
+    t('more'),
+    t('Apply'),
+    t('Reset'),
+    t('Sort by'),
+    t('Asc'),
+    t('Desc'),
+    t('Collection'),
+    t('Object'),
+    t('Collection Object'),
+    t('Object Collection Access'),
+    t('Label'),
+    t('PID'),
+    t('Log ID'),
+    t('.'),
+    t(','),
+    t('All'),
+    t('Block'),
+    t('Usage Stats for Collections'),
+  );
```

Comment:
This function should be returning an associative array of these built up "view" objects. Maybe look at [islandora_batch](https://github.com/Islandora/islandora_batch) for [an example](https://github.com/Islandora/islandora_batch/blob/7.x/views/islandora_batch.views_default.inc#L11-L462)?